src/main/java/com/onarandombox/MultiverseCore/commands/GamerulesCommand.java

```diff
@@ -61,35 +67,46 @@ public void runCommand(CommandSender sender, List<String> args) {
             return;
         }
 
-        final World world;
+        // Not the best way, need to fix with ACF soon...
         if (args.size() == 0) {
             world = p.getWorld();
+        } else if (args.size() == 1) {
+            world = Bukkit.getWorld(args.get(0));
+            if (world == null) {
+                if (p == null) {
+                    sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " World " + ChatColor.AQUA + args.get(0)
+                            + ChatColor.WHITE + " does not exist.");
+                    return;
+                }
+                world = p.getWorld();
+                filter = RegexContentFilter.fromString(args.get(0));
+            }
         } else {
             world = Bukkit.getWorld(args.get(0));
             if (world == null) {
                 sender.sendMessage(ChatColor.RED + "Failure!" + ChatColor.WHITE + " World " + ChatColor.AQUA + args.get(0)
                         + ChatColor.WHITE + " does not exist.");
                 return;
             }
+            filter = RegexContentFilter.fromString(args.get(1));
         }
 
-        ContentDisplay.forContent(getGameRuleMap(world))
-                .header("=== Gamerules for %s%s%s ===", ChatColor.AQUA, world.getName(), ChatColor.WHITE)
-                .colorTool(ColorAlternator.with(ChatColor.GREEN, ChatColor.GOLD))
-                .setting(MapDisplaySettings.OPERATOR, ": ")
-                .show(sender);
+        ContentDisplay.create()
+                .addContentParser(MapContentParser.forContent(getGameRuleMap(world))
+                        .withKeyColor(ChatColor.GREEN)
+                        .withValueColor(ChatColor.YELLOW))
+                .withSendHandler(InlineSendHandler.create()
+                        .withHeader("====[ Gamerules for %s%s%s ]====", ChatColor.AQUA, world.getName(), ChatColor.WHITE)
+                        .withFilter(filter))
+                .send(sender);
     }
 
     private Map<String, Object> getGameRuleMap(World world) {
         Map<String, Object> gameRuleMap = new HashMap<>();
-        for (GameRule<?> rule : GameRule.values()) {
+        Arrays.stream(GameRule.values()).forEach(rule -> {
```

Comment:
Why stream? Generally the overhead isn't worth the cost for small data sets I think.

Reply:
yea, probably dont need to change.